### ISSUE TITLE
pytorch之model.eval | 你说你要一场革命

### ISSUE THREAD

**Issue by Inspiring26** [Inspiring26/Inspiring26.github.io#72]
https://inspiring26.github.io/2019/10/18/pytorch%E4%B9%8Bmodel-eval/ 

 在测试模型是需要用model.eval()使模型变为测试模式，主要是针对dropout和batch normalization在训练时和测试时不一样的情况 其实这么看来，加不加都可以，一般还是加上。
